### ISSUE TITLE
[$4000] Logo shifts to bottom (Bootsplash) when launched - reported by Ekram Khan

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#8236]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app

## Expected Result:
It shifts to bottom

## Actual Result:
It should not shift

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Android (Android 10 (POCO X3)

**Version Number:** 1.1.40-0
**Reproducible in staging?:** need repro
**Reproducible in production?:** need repro 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 

https://user-images.githubusercontent.com/43995119/159107688-15314973-b576-4a33-be72-d243a4dfb309.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @emrankhan & @aneequeahmad
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1644913657527229

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mvtglobally** [comment]
Additional info here 
https://expensify.slack.com/archives/C01GTK53T8Q/p1660111267974929 

https://user-images.githubusercontent.com/43995119/187112643-ae62348e-0587-452c-a6a3-ecfcdae2c1f8.MP4

https://user-images.githubusercontent.com/43995119/187112654-6d4c19cf-4d8d-4d14-a6d0-15379231cd8d.mp4

**Comment by strepanier03** [comment]
I can recreate it, except my logo (on android) shifts down and then back up so it looks like it bounces. My app loads so quickly that I can't capture it with screens either. 

Either way, it's reproducible and the videos above illustrate the behavior well. Sending on.

**Comment by NicMendonca** [comment]
Job posting: https://www.upwork.com/jobs/~01bf7b064475150eec

**Comment by Santhosh-Sellavel** [comment]
Not overdue @Luke9389 
I see `Not a Proirity` label, Can we make this `monthly` instead of `weekly`

cc: @NicMendonca

**Comment by Luke9389** [comment]
Hmm. It looks like @MelvinBot is automatically adding the Daily label (I'm not 100% sure why, but my guess is because we have the `Help Wanted` label on this).

**Comment by NicMendonca** [comment]
@Luke9389 it's because we have a [new proposal](https://github.com/Expensify/App/issues/8236#issuecomment-1248005758)!

**Comment by Luke9389** [comment]
OOoooh I see. Thanks for that context @NicMendonca. I didn't realize that was a thing!

**Comment by NicMendonca** [comment]
@Santhosh-Sellavel would you like to review @aimane-chnaif's proposal?

**Comment by Santhosh-Sellavel** [comment]
@aimane-chnaif 

Are you proposing to make both` Solution 1 & 2` changes https://github.com/Expensify/App/issues/8236#issuecomment-1248005758?

**Comment by Santhosh-Sellavel** [comment]
What do these snippets do?

1:
```xml 
<item name="android:windowTranslucentStatus">true</item>
```

2:
```js
StatusBar.setTranslucent(true);
```

Also, the second snippet seems to be a solution for another bug. We should make we don't make any regression.

**Comment by Santhosh-Sellavel** [comment]
@sobitneupane 

This was added by you, can you recall for which issue?
https://github.com/Expensify/App/blob/31287b661682b5be68ea7ed18ed9ab81e30c933a/src/components/CustomStatusBar/index.android.js#L12

**Comment by Santhosh-Sellavel** [comment]
> So we need to pick up Solution 1 with fixing semi transparent to completely transparent

Can you post an updated proposal?

**Comment by Santhosh-Sellavel** [comment]
Not overdue

**Comment by sobitneupane** [comment]
> @sobitneupane
> 
> This was added by you, can you recall for which issue?
> 
> https://github.com/Expensify/App/blob/31287b661682b5be68ea7ed18ed9ab81e30c933a/src/components/CustomStatusBar/index.android.js#L12

@Santhosh-Sellavel It was added for https://github.com/Expensify/App/issues/7075

**Comment by Santhosh-Sellavel** [comment]
@aimane-chnaif 
Why this?
> * set status bar translucent in native
>   https://github.com/Expensify/App/blob/93ca0413d241c253f4c6e297b7318dd98eebfc8e/android/app/src/main/java/com/expensify/chat/MainActivity.java#L52-L59
>   
>   Insert these lines inside `onCreate` function
> 
> ```diff
>      BootSplash.init(R.drawable.bootsplash, MainActivity.this); // <- display the generated bootsplash.xml drawable over our MainActivity
> +
> +    // make status bar translucent
> +    getWindow().getDecorView().setOnApplyWindowInsetsListener((v, insets) -> {
> +      WindowInsets defaultInsets = v.onApplyWindowInsets(insets);
> +      return defaultInsets.replaceSystemWindowInsets(
> +              defaultInsets.getSystemWindowInsetLeft(),
> +              0,
> +              defaultInsets.getSystemWindowInsetRight(),
> +              defaultInsets.getSystemWindowInsetBottom()
> +      );
> +    });
> ```

Over this?

```xml
<item name="android:windowTranslucentStatus">true</item>
```

**Comment by Santhosh-Sellavel** [comment]
Can we bump this to weekly/monthly @NicMendonca

**Comment by Santhosh-Sellavel** [comment]
Can you elaborate on what the code does?

**Comment by Santhosh-Sellavel** [comment]
> NOTE: `translucent` and `transparent` is different concept `translucent` means no padding under the status bar so content height is same as device height `transparent` means no background color

I get that!

**Comment by Santhosh-Sellavel** [comment]
> ```
>             // If the status bar is translucent hook into the window insets calculations
>             // and consume all the top insets so no padding will be added under the status bar.
> ```

> no padding will be added under the status bar.

What padding is under the status bar now?

**Comment by NicMendonca** [comment]
@Santhosh-Sellavel yep, will change to weekly. But whenever a new proposal is submitted, it'll change back to daily

**Comment by Luke9389** [comment]
Looks like we're still waiting on a proposal that hits the mark here.

**Comment by NicMendonca** [comment]
I doubled the price: https://www.upwork.com/jobs/~01431147666a4d1359

But @Luke9389 @Santhosh-Sellavel let me know if we should close this (based off the `not a priority` label)

**Comment by Luke9389** [comment]
Melvin added that label a while ago, and I think we've since bumped this back up to `Weekly` so I'm taking the `Not a priority` label off.

**Comment by NicMendonca** [comment]
Doubled price: https://www.upwork.com/jobs/~01431147666a4d1359

**Comment by NicMendonca** [comment]
doubled again: https://www.upwork.com/jobs/~01431147666a4d1359

**Comment by Justicea83** [comment]
I have a fix for this, so I'm snapping it

**Comment by NicMendonca** [comment]
@Justicea83 thanks for letting us know! Does this mean we should be opening this up for contributors? 


If so, I doubled the price again: https://www.upwork.com/jobs/~01431147666a4d1359

**Comment by Luke9389** [comment]
@aimane-chnaif, the thing I don't love about this solution is that we probably _shouldn't_ be hiding the status bar. It's got the time, internet connection status, and several other useful things users may want. 

But I'm not sure, do we always hide it when the app is open? No, right?

**Comment by Luke9389** [comment]
OK I think this proposal looks good. After seeing the before and after, it looks like it works just fine. Are you good with this @Santhosh-Sellavel?

**Comment by Santhosh-Sellavel** [comment]
I feel this is more like a workaround.

As explained [here](https://github.com/Expensify/App/issues/8236#issuecomment-1261420574) 

> The way of setting translucent status bar is different between native (java) and javascript.
> ```diff
> +    // make status bar translucent
> +    getWindow().getDecorView().setOnApplyWindowInsetsListener((v, insets) -> {
> +      WindowInsets defaultInsets = v.onApplyWindowInsets(insets);
> +      return defaultInsets.replaceSystemWindowInsets(
> +              defaultInsets.getSystemWindowInsetLeft(),
> +              0,
> +              defaultInsets.getSystemWindowInsetRight(),
> +              defaultInsets.getSystemWindowInsetBottom()
> +      );
> +    });
> ```
> This code is same as StatusBar.setTranslucent(true); call in javascript. (reference: https://github.com/facebook/react-native/blob/main/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java#L114-L152) which makes status bar look translucent. 

We are already calling ` StatusBar.setTranslucent(true)` [here](https://github.com/Expensify/App/blob/9e3a5abef512f7bb7c56993aa3f8474d788f1e5b/src/components/CustomStatusBar/index.android.js#L12)
https://github.com/Expensify/App/blob/9e3a5abef512f7bb7c56993aa3f8474d788f1e5b/src/components/CustomStatusBar/index.android.js#L12

So why we need to add that code is unclear!

**Comment by Santhosh-Sellavel** [comment]
Are there any other issues or supporting docs to understand this better, why should we do this?

> This is called in js level
> 
> This is current issue:
> 
> status bar is not translucent when launch app (before js bundle)
> after call StatusBar.setTranslucent(true) in js, status bar set to translucent (after js bundle)
> so translucent status is different between before js and after js, which makes vertically centered logo shifts because of status bar height
> My solution is to set translucent true natively (before js bundle) to be consistent with after js bundle.
> 
> StatusBar.setTranslucent(true)in js finally runs this native code:

**Comment by Julesssss** [comment]
@aimane-chnaif you're about 80% there with your proposals. I was going to edit your proposal, but I ended up implemented the fix while debugging the issue. 

Native Android uses two Activities. Each Activity has a Theme, and the BootSplash theme is being modified by the `StatusBar.setTranslucent(true)`, as you pointed out: 
- BootSplashActivity | BootTheme --> shows the splash screen on launch
- MainActivity | AppTheme --> container for the entire app

As you said above, we need to prevent the boot splash style from changing, but instead of making it translucent, we [have to use the fitSystemWindows attribute](https://github.com/Expensify/App/pull/12642).  

@aimane-chnaif lets figure out a fair payment % when I'm back tomorrow

**Comment by strepanier03** [comment]
I am out of the office tomorrow. If there is an urgent need on this GH from a BugZero team member, please post in #expensify-open-source for assistance.

**Comment by Julesssss** [comment]
Hey @aimane-chnaif, I've been too busy since last week to take another look but I think the status bar color difference is pretty minor and far better than the more obvious, janky icon. This should still be solvable though 🤞

**Comment by strepanier03** [comment]
Back from ooo and all caught up. Just a friendly bump @Julesssss about the comments from @aimane-chnaif. Will check in tomorrow for actionable steps on my end.

**Comment by Julesssss** [comment]
Now that I understand the React Native [code](https://github.com/facebook/react-native/blob/1453ef1a8836ead03f66792bd36bfcde333434c0/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java#L131-L147) better I'm coming around to [this](https://github.com/Expensify/App/issues/8236#issuecomment-1309660158) solution. I believe the reason for the status colour is that the Activity treats our pages as Modals, instead of new pages -- which by default tints the status bar (and any other visible Views) to highlight the overlaying page. 

I'd like to spend a little bit longer looking into an alternative as I'm still a tiny bit hopeful that we can solve this with a native attribute. Will report back soon.

**Comment by Julesssss** [comment]
I found a solution for the side effects mentioned [here](https://github.com/Expensify/App/issues/8236#issuecomment-1310504297). Working on this again tomorrow.

**Comment by aneequeahmad** [comment]
### PROPOSAL

**PROBLEM**

If existing splash screen is implemented using a dedicated Activity, launching app on devices running Android 12 or higher results in duplicate splash screens: [the new system splash screen](https://developer.android.com/about/versions/12/features/splash-screen) displays, followed by your existing splash screen activity.

You can prevent these degraded or unintended experiences by completing the migration process described in this guide. Once you migrate, the new API improves startup time, gives you full control over the splash screen experience, and ensures a more consistent launch experience with other apps on the platform.

**SOLUTION**

We need to migrate an existing splash screen to the new API(SplashScreen API but google strongly recommend using Androidx SplashScreen compat library). The compat library uses the SplashScreen API, enables backward-compatibility, and creates a consistent look and feel for splash screen display across all Android versions as said by `google developers android doc` [here](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#:~:text=You%20can%20use,look%20and%20feel.)

**Comment by aneequeahmad** [comment]
The official doc has been last updated a few days back [`Last updated 2022-11-09 UTC`](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#:~:text=Last%20updated%202022%2D11%2D09%20UTC.)

**Comment by Julesssss** [comment]
@aneequeahmad your proposal is not clear, as pointing to native code doesn't explain how we will apply changes to our codebase (especially the RN parts). Additionally, the library we're using [already implements](https://github.com/zoontek/react-native-bootsplash/releases/tag/4.3.0) the new Android splash screen library. So this isn't a valid solution. 

The solution I'm working on simply requires an Android attribute change, removal of a library function, and some further page style changes. To be clear, [this regression](https://github.com/Expensify/App/issues/8236#issuecomment-1310504297) is simply due to the fact that our background colours differ between pages -- therefore highlighting the true issue, rather than being a bug itself. The full solution can be achieved via styling, and while I'll leave this open to alternative solutions, any solution from here on mentioning page styling won't count for the bounty -- as I have just given the solution.

**Comment by Julesssss** [comment]
I'm currently speaking with design, as the simplest fix is to stop using a different page background colour. The problem is that web/desktop kinda needs this to differentiate the LHN from chat.

**Comment by strepanier03** [comment]
Thank you for the update @Julesssss. I'll keep following along on this until I can action something.

**Comment by JmillsExpensify** [comment]
@Julesssss To help clear out the extensive backlog of /App bugs, we're putting the spotlight on all bugs older than 4 weeks old. To help unblock the roadmap and get our bug pipeline back in equilibrium, can you:

Decide whether any proposals currently meet our guidelines and can be approved as-is
For any that can't, please take this issue internal and treat it as one of your highest priorities
If you have any questions, don't hesitate to start a discussion in #bug-zero
Thanks everyone!

**Comment by aneequeahmad** [comment]
@Julesssss I have implemented the changes in our code base. I'll post the code changes in a bit. It is working as expected and also add's more features/functionality like animations, delays if need to be added in showing splash screen and other built-in features.

Also my proposal doesn't require any major breaking changes just a few lines added in `android files` to make the splash screen work.

I have seen It is almost like the [react-native-bootsplash](https://github.com/zoontek/react-native-bootsplash) but these changes doesn't require any additional library just followed the doc as i mentioned above and its working like a charm.

cc: @JmillsExpensify

**Comment by Julesssss** [comment]
@aneequeahmad this is still a more complex solution than simply using the [correct android attributes](https://github.com/Expensify/App/pull/12642/files) and it defeats the point of using a single library for boot splash on all platforms. If we hadn't yet implemented boot splash I would agree with you.

**Comment by Julesssss** [comment]
PR is still in progress. Our hover themes don't appear to be configured correctly for the LHNOptionRow component, which is adding a bit of work to this task.

**Comment by Santhosh-Sellavel** [comment]
Not overdue, @Julesssss do we have the PR ready?

**Comment by dylanexpensify** [comment]
@Julesssss just confirming you're leading the fix for this?

**Comment by Julesssss** [comment]
Unfortunately, we seem to be stuck in this discussion. There is no perfect solution and we need to decide between [these options](https://expensify.slack.com/archives/C01GTK53T8Q/p1669112913613109?thread_ts=1668650900.944219&cid=C01GTK53T8Q) before I can finish up the PR and submit.

**Comment by dylanexpensify** [comment]
Should we make this one internal then @Julesssss?

**Comment by Julesssss** [comment]
Not yet because depending on the outcome of that thread we may go with one of the earlier proposals.

**Comment by strepanier03** [comment]
I am out of the office Wednesday-Sunday and will return Monday. If urgent help is needed for this GH please post your request and the URL in #expensify-open-source.

**Comment by tjferriss** [comment]
Following instructions for the [weekly update chore](https://stackoverflow.com/c/expensify/questions/8208):

@Julesssss this is one of the oldest issues in the /App repo. To help us clear out the large backlog of bugs, can you:
- Decide whether any proposals currently meet our guidelines and can be approved as-is
- For any that can't, please take this issue internal and treat it as one of your highest priorities
- If you have any questions, don't hesitate to start a discussion in #bug-zero

**Comment by furkanuzundev** [comment]
# Proposal

At first boot, the logo jumps as the screen is resized again when creating the android status bar.

# Workaround

When we add this code into styles.xml

```javascript
diff --git a/android/app/src/main/res/values/styles.xml b/android/app/src/main/res/values/styles.xml
index 0cad8b818b..9144a0a683 100644
--- a/android/app/src/main/res/values/styles.xml
+++ b/android/app/src/main/res/values/styles.xml
@@ -11,6 +11,7 @@
         <item name="android:statusBarColor">@color/white</item>
         <item name="android:windowLightStatusBar" tools:ignore="NewApi">true</item>
         <item name="android:editTextBackground">@drawable/rn_edit_text_material</item>
+        <item name="android:windowFullscreen">true</item>
     </style>
```

bootsplash will ignore `SOFT_INPUT_ADJUST_RESIZE` and the jump will not occur.

Reference: https://developer.android.com/reference/android/view/WindowManager.LayoutParams#FLAG_FULLSCREEN


https://user-images.githubusercontent.com/30953070/203633345-ae8108ef-0206-47db-b17f-1c4b62a30d21.mov

**Comment by furkanuzundev** [comment]
We are not **hiding**, we are **ignoring** the **resizing** when application created the status bar.

**Comment by Santhosh-Sellavel** [comment]
@Julesssss [This](https://github.com/Expensify/App/issues/8236#issuecomment-1325579507) kind off looks better than now

**Comment by Julesssss** [comment]
We're still discussing [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1669292590231629?thread_ts=1668650900.944219&cid=C01GTK53T8Q), but the dark mode merge which is about to take place makes the mismatching background far worse in my opinion: 

https://user-images.githubusercontent.com/10736861/203783039-438c6f9d-51e0-47af-a46d-16e51b2d8897.mp4

The unique LHN colour is now far more noticable on mobile and looks out of place: 

![Untitled](https://user-images.githubusercontent.com/10736861/203784345-aee9a16e-b4ed-4ee9-900b-bf61f72efde0.png) ![Untitled 2](https://user-images.githubusercontent.com/10736861/203784350-dc3a21eb-4b0c-4ed5-a487-1ecc9649c77c.png) ![Untitled 3](https://user-images.githubusercontent.com/10736861/203784353-db11aa14-99e0-478b-817d-8110f80fb58b.png)

**Comment by furkanuzundev** [comment]
# Proposal

Hello,

I'm sure we're all familiar with the process, but I'll share the steps to explain how I handled the problem.

First of all, we can say that the problem is related to the StatusBar component of react-native, but unfortunately it is very difficult to reach the solution from here.
To understand what's going on, I gave the boot screen a native background and it looks like this;

https://user-images.githubusercontent.com/30953070/203793443-a7872e80-42f2-4758-acaa-ac2387af1d68.mov


When `setTranslucent(true)` while rendering `<CustomStatusBar />`, the app will draw under the status bar. But since it does this at application launch, the screen is moved behind the status bar when it was already under the status bar.

### Workaround

My solution includes a native code snippet.

```java
WindowCompat.setDecorFitsSystemWindows(getWindow(), true);
```
 I say that the application should stay behind the Status Bar while creating the application, by setting the second parameter to true, the application that is behind the Status Bar will fit the screen.(It doesn't affect phone navigation buttons)
 
 Here's all changes.
 
```diff
diff --git a/android/app/src/main/java/com/expensify/chat/MainActivity.java b/android/app/src/main/java/com/expensify/chat/MainActivity.java
index 24dcf36a17..3dfc17a973 100644
--- a/android/app/src/main/java/com/expensify/chat/MainActivity.java
+++ b/android/app/src/main/java/com/expensify/chat/MainActivity.java
@@ -2,6 +2,7 @@ package com.expensify.chat;
 
 import android.os.Bundle;
 import android.content.pm.ActivityInfo;
+import androidx.core.view.WindowCompat;
 import com.expensify.chat.bootsplash.BootSplash;
 import com.facebook.react.ReactActivity;
 import com.facebook.react.ReactActivityDelegate;
@@ -52,6 +53,7 @@ public class MainActivity extends ReactActivity {
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(null);
+    WindowCompat.setDecorFitsSystemWindows(getWindow(), true);
     if (getResources().getBoolean(R.bool.portrait_only)) {
       setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
     }
diff --git a/src/components/CustomStatusBar/index.android.js b/src/components/CustomStatusBar/index.android.js
index 464d743739..7d42eaf937 100644
--- a/src/components/CustomStatusBar/index.android.js
+++ b/src/components/CustomStatusBar/index.android.js
@@ -9,7 +9,8 @@ export default class CustomStatusBar extends React.Component {
     componentDidMount() {
         StatusBar.setBarStyle('dark-content');
         StatusBar.setBackgroundColor('transparent');
-        StatusBar.setTranslucent(true);
+
+        // StatusBar.setTranslucent(true);
     }
 
     render() {
```

### Result


https://user-images.githubusercontent.com/30953070/203794680-4b8bc612-2f93-4680-80e3-43eff6021f48.mov


Also, i have checked on real device @aimane-chnaif, it look's same.

**Comment by Julesssss** [comment]
Hi @furkanuzundev, thanks for the proposal, but it is the same solution as this already shared [PR](https://github.com/Expensify/App/pull/12642/files). The only difference is that instead of using the attribute (which is preferred), we're programmatically setting the flag.

**Comment by furkanuzundev** [comment]
Thanks @Julesssss, I overlooked that.Is the solution in this PR accepted?

@aimane-chnaif 
When we call `WindowCompat.setDecorFitsSystemWindows(getWindow(), true);` in **onCreate** method, we are creating the activities with attributes.
If we do this on the react-native side,  we have made an update after the attributes have been created.

**Comment by Julesssss** [comment]
Yeah @furkanuzundev, I'm just about to share the plan.

**Comment by furkanuzundev** [comment]
Do you see that nothing changes when you add  `WindowCompat.setDecorFitsSystemWindows(getWindow(), true);` ?

**Comment by furkanuzundev** [comment]
Yes, absolutely no change is visible.

And as you said, `StatusBar.setTranslucent(true)` already does the same job.

Since it has been a topic that has been discussed for a long time, I have searched for a new solution, rather than the solution in front of my mind psychologically.

Seriously, why did it take so long when the answer was so clear?

**Comment by Julesssss** [comment]
Alright, I'm going to go ahead and merge the splash fix once I make some final improvements [PR](https://github.com/Expensify/App/pull/12642/files) that uses some suggestions from @aimane-chnaif's proposals. I opened this [follow-up issue](https://github.com/Expensify/App/issues/13001) which we'll use to continue discussing the fix for Android's now-obvious status bar.

**Comment by Julesssss** [comment]
[PR ](https://github.com/Expensify/App/pull/12642/files)has been updated to align with the upcoming dark mode changes. While it works well, I'm holding until that has been merged so that I can guarentee the changes are compatable. 

The dark-mode PR is large enough that testing everything will be tricky.

**Comment by Julesssss** [comment]
[The PR](https://github.com/Expensify/App/pull/12642) contains screenshots for every platform in both light and dark mode.

**Comment by strepanier03** [comment]
Back from OoO and will continue to follow along with the progress here until I can take action. @Julesssss let me know if there is anything else I can do to help in the meantime.

**Comment by Julesssss** [comment]
> ⚠️ Looks like this issue was linked to a **Deploy Blocker** [here](https://github.com/Expensify/App/issues/13111)
> 
> If you are the assigned CME please investigate whether the linked PR caused a regression and leave a comment with the results.
> 
> If a regression has occurred and you are the assigned CM follow the instructions [here](https://stackoverflow.com/c/expensify/questions/13647).
> 
> If this regression could have been avoided please consider also proposing a recommendation to the PR checklist so that we can avoid it in the future.

No action required. The linked issue is already fixed with a separate PR

**Comment by strepanier03** [comment]
I am OoO for jury duty tomorrow. If immediate action is needed on this GH please post in [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q) for another BZ's assistance, thank you.

**Comment by Santhosh-Sellavel** [comment]
@Julesssss I feel I'm eligible for some compensation here too.

**Comment by Julesssss** [comment]
Yeah, sorry. I assumed that was implied by your assignment. 

@strepanier03 please pay 3/4 of the bounty plus a PR payment for @aimane-chnaif and a C+ payment for @Santhosh-Sellavel.

**Comment by ekramkhan66** [comment]
@strepanier03 I think I am also eligibility for payment as a issue reporter. Thanks!

**Comment by strepanier03** [comment]
I can't interact with the previous job that was created for this GH. I'm going to create a new one now and share links soon.

**Comment by strepanier03** [comment]
@Santhosh-Sellavel and @aimane-chnaif - Please apply for the job [HERE](https://www.upwork.com/jobs/~014b777006c5d071de) and let me know when you do so I can go in and accept/pay. 

@ekramkhan66 - Please apply for the reporting payment [HERE](https://www.upwork.com/jobs/~010625318be1833f21) and let me know when you do so I can accept/pay.

**Comment by strepanier03** [comment]
- [x] TestRail @strepanier03: Not creating a TR for this edge case.
- [ ] Pay @aimane-chnaif $3000
- [ ] Pay @Santhosh-Sellavel $3000
- [ ] Pay @ekramkhan66 $250

**Comment by ekramkhan66** [comment]
@strepanier03 applied. Thanks!

**Comment by strepanier03** [comment]
@aneequeahmad can you please share a Slack link for where you reported this? I have the link in the OG comment but it's to a report from Ekram Khan.

**Comment by aneequeahmad** [comment]
@strepanier03 here is the link to the slack bug report https://expensify.slack.com/archives/C01GTK53T8Q/p1660111267974929

**Comment by strepanier03** [comment]
Thank you for that link @aneequeahmad. 

After looking at this more closely it looks like you reported on August 9th and it was marked a dupe:
<img width="758" alt="2022-12-08_19-34-27" src="https://user-images.githubusercontent.com/10925636/206618521-1bc59045-de7a-4701-a8da-c2b567c4463f.png">

The original report by @ekramkhan66 was done on February 15:
<img width="748" alt="2022-12-08_19-34-47" src="https://user-images.githubusercontent.com/10925636/206618596-f087f84c-76f4-4aa8-8e54-2950894aad94.png">

**Comment by Santhosh-Sellavel** [comment]
Thanks, @strepanier03!

**Comment by aneequeahmad** [comment]
@strepanier03 Actually this bug wasn't reproducible and was closed when intially reported.

- Also, the bug i reported is that `Logo shifts up when launched` and i believe this is the bug that was there as confirmed by others here(https://github.com/Expensify/App/issues/8236#issuecomment-1237791427)
 
- I reported that the logo was shifting upwards when launched and then i was added by the @mvtglobally in the bug reporters 
section which indicates that at that time i was eligible or otherwise i wouldn't have been added in the OP.

- It was marked as dupe before i was assigned in the bug reported section.

**Comment by strepanier03** [comment]
Thank you @aneequeahmad, I've raised this internally to get some agreement and I'll update you once that convo wraps up, sorry for the delay!

**Comment by strepanier03** [comment]
@aneequeahmad - Please apply for the job [here](https://www.upwork.com/jobs/~01230b4cac831b2de0) and I'll check tomorrow to accept and send the contract back to you. Cheers!

**Comment by shoaibarif244** [comment]
@mvtglobally 
The requirement is to move the logo toward the bottom of the splash screen when the user opens the app.
[**Solution**]: I'll fix this issue using the Animated Library of react-native.

**Comment by shoaibarif244** [comment]
@mvtglobally @MelvinBot Please inform me if you accept my proposal so that I could apply for the job on **Upwork**.

**Comment by Julesssss** [comment]
@shoaibarif244 this issue has already been resolved, please review the [contributing guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#finding-jobs).

**Comment by strepanier03** [comment]
Waiting for final pay out for @aneequeahmad and then ready to close.

**Comment by aneequeahmad** [comment]
@strepanier03 I applied for the job. Thank you for mentioning

**Comment by strepanier03** [comment]
Great thank you @aneequeahmad, I have accepted it and will check tomorrow to pay the contract. I am offline the rest of the day unexpectedly.

**Comment by strepanier03** [comment]
Final payment made, going to close this one out now. Thank you to all involved!

**Comment by ekramkhan66** [comment]
I just tested the latest version, and it seems like the issue is not fixed. 

Android (Poco X3) 

https://user-images.githubusercontent.com/99305361/208656076-b135a672-2c40-4cf9-9713-09bdf1d02adc.mp4

**Comment by Julesssss** [comment]
> I just tested the latest version, and it seems like the issue is not fixed.

This is a specific device, with a custom Android ROM. It's a separate issue, and most likely one that we won't prioritize solving at the moment as it occurs on a very small fraction of user devices.
